docs/components/multiselect/index.md

```diff
@@ -0,0 +1,308 @@
+</Form::Controls::Multiselect>
+```
+
+```js
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked selected;
+
+  options = ['Blue', 'Red', 'Yellow'];
+
+  @action
+  handleChange(option) {
+    this.selected = option;
+  }
+}
+```
+
+## Option Key
+
+Optional.
+
+The `@optionKey` argument is used when your `@options` take the shape of an array of objects. The `@optionKey` is used to determine two things internally:
+
+1. The displayed value inside of each selected chip of the multiselect
+2. Used as the key in the default filtering scenario where we filter `@options`. To properly filter the `@options` based on the user input from the textbox, we need to know how to compare the entered value to each object. The `@optionKey` tells us which key of the object to use for this filtering.
+
+In the example below, we set `@optionKey='label'`. Our `@options` objects have a `label` key and we want the label of the selected option to be used for the selected value, as well as for filtering as the user types.
+
+```hbs
+<Form::Controls::Multiselect
+  @onChange={{this.handleChange}}
+  @options={{this.options}}
+  @optionKey='label'
+  @selected={{this.selected}}
+  as |multiselect|
+>
+  <multiselect.Option>
+    {{multiselect.option.label}}
+  </multiselect.Option>
+</Form::Controls::Multiselect>
+```
+
+```js
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked selected;
+
+  options = [
+    {
+      label: 'Blue',
+      value: 'blue',
+    },
+    {
+      label: 'Green',
+      value: 'green',
+    },
+    {
+      label: 'Yellow',
+      value: 'yellow',
+    },
+    {
+      label: 'Orange',
+      value: 'orange',
+    },
+    {
+      label: 'Red',
+      value: 'red',
+    },
+    {
+      label: 'Purple',
+      value: 'purple',
+    },
+    {
+      label: 'Teal',
+      value: 'teal',
+    },
+  ];
+
+  @action
+  handleChange(option) {
+    this.selected = option;
+  }
+}
+```
+
+## Remove Button Labels
+
+Toucan Core wants to provide accessible components, so a component argument of `@removeButtonLabelFunction` is exposed. This component argument is used to set the `aria-label` of each selected chip's remove button. This ensures screenreader users have all of the context they need when remove items from the selected list. The current option is returned to you so that you can format the string as you please. This comes in handy as well when dealing with translated strings.
+
+```hbs
+<Form::Controls::Multiselect
+  @removeButtonLabelFunction={{this.removeButtonLabelFunction}}
+/>
+```
+
+```js
+removeButtonLabelFunction(option) {
+  // Or if dealing with objects, this may be something like:
+  // return `Remove ${option.label}`;
+  return `Remove ${option}`;
+}
+```
```

Comment:
This is somewhat lame, but we are required to ensure these strings can be translated.  A function callback was the only way I could see this working, where we can pass back the option and the consumers can format it as needed.  Totally open to other suggestions though!